src/code/z_play.c

```diff
@@ -436,10 +436,10 @@ void Play_Init(GameState* thisx) {
     AnimationContext_Update(this, &this->animationCtx);
     gSaveContext.respawnFlag = 0;
 
-    if (dREG(95) != 0) {
-        D_8012D1F0 = D_801614D0;
-        osSyncPrintf("\nkawauso_data=[%x]", D_8012D1F0);
-        DmaMgr_DmaRomToRam(0x03FEB000, D_8012D1F0, sizeof(D_801614D0));
+    if (R_USE_DEBUG_CUTSCENE) {
+        gDebugCutsceneScript = sDebugCutsceneScriptBuf;
+        osSyncPrintf("\nkawauso_data=[%x]", gDebugCutsceneScript);
+        DmaMgr_DmaRomToRam(0x03FEB000, gDebugCutsceneScript, sizeof(sDebugCutsceneScriptBuf));
```

Comment:
Can anything be done about this hardcoded address `0x03FEB000`? Is there a symbol or something?

Reply:
Welp lmao. Might warrant a comment then.

Reply:
I dont think its a bug, just leftover debugging stuff. this will never run unless you tell it to (and if you do want to use this debug feature, you should point it to your own data)

imo the hardcoded address speaks for itself, but if we really want to comment on it i can

Reply:
i added a comment